src/roman.rs

```diff
@@ -144,7 +146,7 @@ impl Roman {
         }
     }
 
-    pub fn into_inner(self) -> u16 {
+    pub fn into_inner(self) -> NonZeroU16 {
```

Comment:
Here, specifically, it seems like it'd be easier just to make the call to `.get()` and return a `u16` instead of having to add `get()` everywhere this is used.

Reply:
I think `into_inner` should stay with `NonZeroU16` as it's the _inner_ type (Also, see previous comment).

But we can add a method like `value` that will return `u16`:
```rust
pub fn value(self) -> u16 {
    self.into_inner().get()
}
```
This will be convenient in both cases — when using `u16` and when using `NonZeroU16`

Reply:
The current version of the code implements:
```
fn get(self) -> u16
```
Maybe just also add
```
fn into_inner(self) -> NonZeroU16
```
?

Likewise relates to previous comment.